diff --git a/Makefile.libretro b/Makefile.libretro
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -596,2 +596,18 @@ else ifeq ($(platform), miyoo)
 
+# RGB30
+else ifeq ($(platform), rgb30)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR = $(CROSS_COMPILE)ar
+	CFLAGS += -mtune=cortex-a55 -march=armv8.2-a
+	CFLAGS += -fomit-frame-pointer -ffast-math -fdata-sections -ffunction-sections -fsingle-precision-constant -flto -fPIC
+	LDFLAGS += -flto -fPIC
+	OPTIMIZE += -Ofast -DNDEBUG=1
+	HAVE_NEON = 1
+	BUILTIN_GPU = neon
+	CPU_ARCH := arm64
+	ARCH = arm64
+	DYNAREC = ari64
+
 # Emscripten
